src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Introduction to variable fonts on the web
+subhead: A new font specification that can significantly reduce font file sizes
+description: >
+  How variable fonts work, how typographers implement variable fonts, and
+  how to work with variable fonts in CSS.
+authors:
+  - mustafakurtuldu
+  - thomassteiner
+date: 2018-02-19
+updated: 2020-05-12
+hero: hero.jpg
+tags:
+  - blog
+  - fonts
+---
+
+Variable fonts can significantly reduce the size of your font files and make
+it possible to animate your font characters. This guide explains how
+variable fonts work, how typographers implement variable fonts, and how
+to work with variable fonts in CSS.
```

Comment:
Rewrote the intro paragraph to make the value proposition of VFs clearer

Reply:
works for me : ), the images seem broken again though, was that what you meant by build not working?